packages/terra-text/tests/nightwatch/text-spec.js

```diff
@@ -0,0 +1,48 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  before: (browser, done) => {
+    browser.resizeWindow(browser.globals.width, browser.globals.height, done);
+  },
+
+  afterEach: (browser, done) => {
+    screenshot(browser, 'terra-text', done);
+  },
+
+  'Displays a default text component': (browser) => {
+    browser.url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/text-tests/default`);
+    browser.expect.element('.terra-Text').text.to.equal('Default');
```

Comment:
Can we add to the tests any class checking to make sure that the text also contains the correct classes used for styling?